pandas/conftest.py

```diff
@@ -921,6 +921,23 @@ def all_binary_operators(request):
     return request.param
 
 
+@pytest.fixture(
```

Comment:
isn't this all_compare_operators( ?

Reply:
we don't really distinguish the naming here.  e.g. for arithmetic we have
`all_binary_operations` (using operator) and `all_arithmetic_operators`. 

so ideally have some good pattern here. 

maybe 
`all_comparison_operators` (use `operator`)
`all_comparison_dunder` (use `dunder`)

would have to fix a bunch of things but prob worth it.

Reply:
yah not something i want to futz with

Reply:
sure but we cannot add it like this, so need to do something. e.g. move next to the others & rename this one to conform.

Reply:
will move this down adjacent to all_compare_operators

Reply:
looks like most of the places here we use the dunder one then do a `getattr(operator, all_comparison_operators.strip("__"))`, so will just replace those usages with this